lib/algfld.gd

```diff
@@ -188,3 +188,45 @@ DeclareGlobalFunction("AlgExtEmbeddedPol");
 
 DeclareGlobalFunction("AlgExtSquareHensel");
 
+#############################################################################
+##
+#F  DecomPoly( <f> [:"onlyone"] )  finds ideal decompositions of rational f
```

Comment:
Open question: should the name of this function stay `DecomPoly`? Wouldn't e.g. `DecompositionsOfPolynomial` more "GAPish"? I don't mind much either way, but once we document the name, we are more or less stuck with it, so I just want this to be a conscious decision.

Reply:
If we want to change it, then `IdealDecompositionsOfPolynomial` would be the longish name.

Reply:
If we document it, I'd kind of prefer the longish name (a shorter alias can still be added). That said, I don't terribly mind if you greatly prefer the shorter name.